reference/functions/readenvfile.markdown

```diff
@@ -0,0 +1,56 @@
+---
+layout: default
+title: readenvfile
+published: true
+tags: [reference, io functions, functions, readenvfile, json, env, os-release, container]
+---
+
+[%CFEngine_function_prototype(filename, maxbytes)%]
+
+**Description:** Parses key-value pairs from the file `filename` in env file format (`man os-release`).
+Returns the result as a `data` variable.
+Keys and values are interpreted as strings.
+`maxbytes` is optional, if specified, only the first `maxbytes` bytes are read from `filename`.
+[Details of the os-release/env file format on freedesktop.org](https://www.freedesktop.org/software/systemd/man/os-release.html)
+
+[%CFEngine_function_attributes(filename, maxbytes)%]
+
+**Example:**
+
+```cf3
+    vars:
+      "loadthis"
+         data => readenvfile("/etc/os-release", 4000);
+```
+
+Complete example, run with `cf-agent -f file.cf`:
```

Comment:
I would simply include the example from core/examples.

For example: https://github.com/cfengine/documentation/blob/master/examples/example-snippets/general.markdown#hello-world and https://docs.cfengine.com/docs/3.10/examples-example-snippets-general.html#hello-world

https://github.com/cfengine/documentation/blob/master/cheatsheet.markdown#symlink-example
https://docs.cfengine.com/docs/master/markdown-cheatsheet.html#symlink-example